modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_media.default.yml

```diff
@@ -0,0 +1,29 @@
+uuid: ec9cb7b3-5791-4ebe-a525-46b9bc34447c
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.